src/Components/Display.js

```diff
@@ -0,0 +1,12 @@
+import React, { Component } from 'react';
+
+export default class Display extends React.Component {
+  render() {
+    const { result } = this.props;
+    return (
+      <div className="Display">
+        {result}
+      </div>
+    );
+  }
+}
```

Comment:
Add prop validation to track all props that we will use in the component

`
Display.defaultProps = {
  result: '0',
};
`